src/mindlessgen/qm/xtb.py

```diff
@@ -42,11 +42,13 @@ def optimize(
             molecule.write_xyz_to_file(str(temp_path / "molecule.xyz"))
 
             # run xtb
+            gfn_level = self.cfg.level
```

Comment:
Why do you initialize a single variable beforehand? Is this necessary? Why not putting `self.cfg.level` directly in the `arguments` definition?

Reply:
To have the variable at hand in case I needed it later.

Reply:
But you have the variable also as an attribute from `self.cfg` at hand? There's no advantage of assigning this to another variable.

Reply:
The changes should now be included.